### PR TITLE
submitting circle_class.py and mailroom_oo.py

### PR DESCRIPTION
**Pull request by cthaehtruthteller**
Submitting Lesson08 assignment

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@cthaehtruthteller can you please submit your assignments on the course website?

also, what is your edx user name or name please? thanks.

**Comment by natasha-aleksandrova** [comment]
@cthaehtruthteller we were able to figure out your username in EdX - you are currently aren't showing as active student in the report, and we are working on resolving this so we can submit the final grade for you.

Apologies for the delay.

**Comment by cthaehtruthteller** [comment]
Thanks for letting me know. I'll keep an eye out for any updates.

File: students/thecthaeh/Lesson08/circle_class.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    @property
+    def area(self):
+        _area = (math.pi * (math.pow(self.radius, 2)))
+        return round(_area, 2)
+    
+    @classmethod
+    def from_diameter(cls, diam_value):
+        cls.diameter = diam_value
+        cls.radius = round((cls.diameter / 2), 2)
+        return cls
```
Comment (natasha-aleksandrova) [inline students/thecthaeh/Lesson08/circle_class.py:41]: you want to use the constructor to create the instance -> `return cls(diam_value / 2.0)`

**Pull request merged**